scipy/stats/_continuous_distns.py

```diff
@@ -4171,6 +4171,126 @@ def _entropy(self, c):
 wrapcauchy = wrapcauchy_gen(a=0.0, b=2*pi, name='wrapcauchy')
 
 
+class gennorm_gen(rv_continuous):
+    """A generalized normal continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `gennorm` is [1]_::
+
+                                     beta
+        gennorm.pdf(x, beta) =  ---------------  exp(|x|**beta)
+                                2 gamma(1/beta)
+
+    `gennorm` takes ``beta`` as a shape parameter.
+    For beta = 1, it is identical to a Laplace distribution.
+    For beta = 2, it is identical to a normal distribution (with scale=1/sqrt(2)).
```

Comment:
Also please use double backticks for the code-like markup: ``scale = 1/ sqrt(2)``.